zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,310 @@
+            b = b.strip()
+            if b == ib:
+                break
+
+        return ib
+
+    @staticmethod
+    def get_upload_param(headers):
+        def get_long_field(key, default=None):
+            v = headers.get(key, default)
+            try:
+                lv = long(v)
```

Ruff (0.13.3)

Undefined name `long`

(F821)